.styleci.yml

```diff
@@ -10,4 +10,4 @@ disabled:
 
 finder:
   exclude:
-    - 'fixtures'
+    - 'reserved_classes.php'
```

Comment:
Exclude not working at present, will need to check or just integrate into the test as a string.